plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -817,6 +819,11 @@ public boolean configure(final String name, final Map<String, Object> params) th
             directDownloadTemporaryDownloadPath = getDefaultDirectDownloadTemporaryPath();
         }
 
+        // Enable/disable IO driver for Qemu (in case it is not set CloudStack can also detect if its supported by qemu)
+        // Do not remove - switching it to AgentProperties.Property may require accepting null values for the properties default value
+        String enableIoUringConfig = (String) params.get(ENABLE_IO_URING_PROPERTY);
+        enableIoUring = isIoUringEnabled(enableIoUringConfig);
```

Comment:
```suggestion
        enableIoUring = isIoUringEnabled(enableIoUringConfig);
```
needs to be moved after this part of the code 
 https://github.com/apache/cloudstack/blob/363a2cff8231a31c3c9def329805910e43358b1d/plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java#L1126-L1133

otherwise it will be always `false`

Reply:
Thanks @slavkap, done